src/transport/PairingSession.h

```diff
@@ -36,6 +36,28 @@ class DLL_EXPORT PairingSession
     PairingSession() {}
     virtual ~PairingSession() {}
 
+    NodeId GetPeerNodeId() const { return mPeerNodeId; }
+    void SetPeerNodeId(NodeId peerNodeId) { mPeerNodeId = peerNodeId; }
+
+    uint16_t GetPeerKeyId() const { return mPeerKeyId; }
+    void SetPeerKeyId(uint16_t id) { mPeerKeyId = id; }
+    bool IsValidPeerKeyId() const { return mPeerKeyId != kInvalidKeyId; }
+
+    uint16_t GetLocalKeyId() const { return mLocalKeyId; }
+    void SetLocalKeyId(uint16_t id) { mLocalKeyId = id; }
+    bool IsValidLocalKeyId() const { return mLocalKeyId != kInvalidKeyId; }
+
+    const Transport::PeerAddress & GetPeerAddress() const { return mPeerAddress; }
+    Transport::PeerAddress & GetPeerAddress() { return mPeerAddress; }
+    void SetPeerAddress(const Transport::PeerAddress & address) { mPeerAddress = address; }
+
+    void Clear()
```

Comment:
Clear should not be public API.  It doesn't make sense to clear the PairingSession but not its subclasses....

Reply:
Added a TODO, should always construct a new instance instead reset the old one